src/components/about/about.jsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import "./about.scss";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import kim from "../../images/kim.jpg";
+import green from "../../images/green.jpg";
+import johnson from "../../images/johnson.jpg";
+//about section
+//button has no function at the moment
+class About extends React.Component {
+  render() {
+    return (
+      <React.Fragment>
+        <section className="about-container">
+          <section className="about-col">
+            <div className="about-col-1">
+              <p>
+                The idea of Nouri Meals was born out of a realization that we
```

Comment:
Add a transition like a header or some divider to let people know they are in the about section.